launch.go

```diff
@@ -239,28 +239,29 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	metaLog := ""
 
 	parentBuildIDs := convertToArray(build.ParentBuildID)
+	mergedMeta := map[string]interface{}{
```

Comment:
should we namespace this under build ?

Reply:
not sure, I feel like top level is better since some people might set `meta.build` themselves and it might be confusing

Reply:
wasn't sure what made more sense for preventing people from accidentally setting a special meta field

for meta status and summary I've nested them so people need to set `meta set meta.summary.coverage` or `meta set meta.status.findbugs`

`meta.build` might be better for consistency